utils/global-state-update-gen/src/auction_utils.rs

```diff
@@ -64,18 +64,18 @@ pub fn find_large_bids(
     builder: &mut LmdbWasmTestBuilder,
     new_snapshot: &SeigniorageRecipientsSnapshot,
 ) -> BTreeSet<PublicKey> {
-    let min_bid = new_snapshot
-        .values()
-        .next()
-        .unwrap()
+    let seigniorage_recipients = new_snapshot.values().next().unwrap();
+    let min_bid = seigniorage_recipients
         .values()
         .map(SeigniorageRecipient::stake)
         .min()
         .unwrap();
     builder
         .get_bids()
         .into_iter()
-        .filter(|(_pkey, bid)| bid.staked_amount() >= min_bid)
+        .filter(|(pkey, bid)| {
+            bid.staked_amount() >= min_bid && !seigniorage_recipients.contains_key(pkey)
```

Comment:
There is also a part of the state that explicitly states who is going to be a validator - but that part is being overwritten by the auction contract at the end of each era based on the bids. So we need to change the bids in order for the auction contract to keep working as intended.

This is basically hacking the auction contract state so that it does what we want it to do.

Reply:
Shouldn't we change the bids of whoever is not the new validator to 0? Or remove them entirely.

Reply:
I guess we could, but this was intended to do the bare minimum necessary, which is to reset the bids of only the non-validators who would become validators when the auction is run. The smaller bids don't affect the validator set, and so it's not necessary to change them.